app/src/main/res/values/strings.xml

```diff
@@ -810,4 +810,9 @@
     <!-- Open source licenses page title
     The first parameter is the app name -->
     <string name="open_source_licenses_title">%s | OSS Libraries</string>
+
+    <!-- Content description of the tab counter toolbar button when one tab is open -->
+    <string name="tab_counter_content_description_one_tab">1 tab</string>
+    <!-- Content description of the tab counter toolbar button when multiple tabs are open. First parameter will be replaced with the number of tabs (always more than one) -->
+    <string name="tab_counter_content_description_multi_tab">%d tabs</string>
```

Comment:
I wish we could use quantity strings...

Reply:
Go ahead and implement it! :D
https://hg.mozilla.org/l10n/compare-locales/

CC @Pike